src/Form/ProfilCommercialType.php

```diff
@@ -81,7 +81,7 @@ public function buildFormLastPart(FormBuilderInterface $builder): void
     {
         $builder
         ->add('numberClosPerMonth', ChoiceType::class, [
-            'label' => 'Combien de closing fetes-vous par mois ?',
+            'label' => 'Combien de contrats fetes-vous par mois ?',
```

Comment:
Maybe without french errors, it would be better ...

Reply:
Sorry